docs/README-AWS.md

```diff
@@ -127,7 +84,7 @@ Resources:
       }
     },
     {
-      "PolicyName": "ElasticServerlessForwarderFunctionRolePolicySQSContinuingQueue", ## ADD AS IT IS FOR THE REPLAY QUEUE
+      "PolicyName": "ElasticServerlessForwarderFunctionRolePolicySQSReplayQueue", ## ADD AS IT IS FOR THE REPLAY QUEUE
       "PolicyDocument": {
```

Comment:
I was wondering if "Fn:GetAtt" on "ElasticServerlessForwarderContinuingQueue" will work in all enviornments?
For instance in my enviornment the queue name is "elastic-serverless-forwarder-continuing-queue-0ee8749bc847". Maybe I don't understand this fully but can we double check please?

In my setup I added a policy via copying ARN.

Reply:
You are right, `Fn:GetAtt` is specific to cloudformation template.
I will split this two policies for cloudformation and console